openforcefield/topology/topology.py

```diff
@@ -502,29 +359,51 @@ def _charge_molecules(self, chargeMethod='AM1-BCC'):
 
 class Particle(object):
     """
-    A particle in a system
+    Base class for all particles in the system.
 
-    This could be an ``Atom`` or a ``VirtualSite``
+    A particle object could be an ``Atom`` or a ``VirtualSite``.
 
     """
-    def __init__(self):
+    def __init__(self, name):
         """
         Create a particle.
         """
-        pass
+        self._name = name
+        self._topology = None # the Topology object this Particle belongs to
+
+    @property
+    def topology(self):
+        """
+        The Topology object that owns this particle, or None.
+        """
+        return self._topology
+
+    @property
+    def name(self):
+        """
+        An arbitrary label assigned to the particle.
+
+        """
+        return self._name
 
+    # TODO: Should this just be index?
     @property
     def particle_index(self):
         """
-        Index of this particle within the ``Topology``
+        Index of this particle within the ``Topology`` or corresponding OpenMM ``System`` object.
 
         .. todo::
 
-           Should this just be called ``index``, or does that risk confusion within
-           the index within ``topology.atoms``?
+           Should ``atom.particle_index`` just be called ``index``, or does that risk confusion within
```

Comment:
I'm OK with `particle_index` but `index` would be confusing. And perhaps even avoiding the `index` word entirely for this would be better; perhaps `particle_number` or `particle_identifier` or something?